packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -234,7 +234,8 @@ export async function getNodeVersion(
 
 export async function scanParentDirs(
   destPath: string,
-  readPackageJson = false
+  readPackageJson = false,
+  corepackEnabled = false
```

Comment:
Lets use `options` here instead because this could turn into a lot more props in the future